packages/gatsby-react-router-scroll/src/ScrollBehaviorContext.ts

```diff
@@ -47,16 +69,22 @@ class ScrollContext extends React.Component {
     this.scrollBehavior.updateScroll(prevRouterProps, { history, location })
   }
 
-  componentWillUnmount() {
+  componentWillUnmount(): void {
+    // Type definitions from scroll-behavior forgot to include this method
+    // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+    // @ts-ignore
     this.scrollBehavior.stop()
   }
 
-  getRouterProps() {
+  getRouterProps(): IContext {
```

Comment:
this feels wrong. iContext is an empty interface

Reply:
Sorry, I will fix that